### DISCUSSION TITLE
Hey someone help me how to make ui like yt 2017

### DISCUSSION THREAD

**Discussion by Sooub** [Rehike/Rehike#625] [General]
Someone help me how to make Ui Like yt 2017 there are so many custom but my main is make like yt 2017

**Comment by ghost** [comment]
YouTube provided two styles in 2017. Which one are you requesting?

![image](https://github.com/user-attachments/assets/9cce117e-116f-4fa9-9a08-ee3083fd2b4d)

**Reply by Sooub** [reply]
left

**Reply by ghost** [reply]
Rehike does not handle this style, however you might find success with StarTube by lightbeam.

**Reply by Sooub** [reply]
Oh and how to make tream dark

**Reply by ghost** [reply]
Tream?

**Reply by Sooub** [reply]
sorry i treble speak is how to make theme dark

**Reply by ghost** [reply]
In Rehike? Rehike doesn't support a dark theme since it wasn't provided as an option by YouTube officially for the frontend we preserve.

**Reply by Sooub** [reply]
how i make ui like yt 2017
pls help

**Reply by ghost** [reply]
I already told you, look into StarTube by lightbeam. You can use Google, can't you?

**Comment by Sooub** [comment]
![Screenshot (1)](https://github.com/user-attachments/assets/635e0734-95a1-4c16-b822-f15cf14c85ec)
it one

**Reply by ghost** [reply]
Yes. You'll need V3 as well.

**Comment by Sooub** [comment]
but im not use v3

**Reply by ghost** [reply]
Then you'll need to switch over to it. What else do you want me to say?

**Reply by Sooub** [reply]
nope but thx and how i gonna uninstall rehike

**Reply by Sooub** [reply]
i gonna switch

**Reply by ghost** [reply]
Remove the Rehike redirection line in hostsfile `127.0.0.1 www.youtube.com`, and, if you want to, uninstall XAMPP.
